c++/src/kj/thread.h

```diff
@@ -59,7 +59,7 @@ class Thread {
     Function<void()> func;
     kj::Maybe<kj::Exception> exception;
 
-    int refcount;
+    unsigned int refcount;
```

Comment:
It's a signed `long`.  The compiler error I got offered an `unsigned int` overload even though I don't see it listed on the MSDN docs.  I figured the latter was a less intrusive change.